include/uv/win.h

```diff
@@ -303,7 +303,8 @@ typedef struct {
   LPFN_ACCEPTEX func_acceptex;
 
 #define uv_tcp_connection_fields                                              \
-  uv_buf_t read_buffer;                                                       \
+  uv_tcp_accept_t* dummy1; /* Mirror of union field, keep as NULL */          \
```

Comment:
It turned out we were reading this field through the overlapping `uv_tcp_server_fields ` struct fields until the connection was established (setting the UV_STREAM_CONNECTION flag). That meant that dropping (or using) this field would cause libuv to crash, so I've kept the object layout the same and just renamed it to "dummy". Any concerns with this approach?

Reply:
Silence is no concern?

Reply:
I believe we have done this before for the same reason, so I don’t think it’s of any concern!